src/Symfony/HmacResponseListener.php

```diff
@@ -20,9 +16,9 @@
 class HmacResponseListener implements EventSubscriberInterface
 {
     /**
-     * @param FilterResponseEvent $event
+     * @param \Symfony\Component\HttpKernel\Event\ResponseEvent $event
```

Comment:
since it's pulled in with a use statement, shouldn't this be
```suggestion
     * @param ResponseEvent $event
```

Reply:
Using the fully qualified class name in Docblocks was a code style decision made when we refactored this to support HTTP HMAC v2, so this is to keep it consistent with the repo's coding standards.